2-mandatory/1-weather-report.js

```diff
@@ -10,54 +10,72 @@
             For example, "The temperature in London is 10 degrees"
         - Hint: you can call the temperatureService function from your function
 */
+const cities = [
+  "London",
+  "Paris",
+  "Barselona",
+  "Dubai",
+  "Mumbai",
+  "São Paulo",
+  "Logos",
+];
 
 function getTemperatureReport(cities) {
-    // TODO
-}
+   const citystrings = [];  
+  
+    for (const item of cities) {
+      citystrings.push(
```

Comment:
Will be nice to see there String Interpolation👌

Reply:
I agree with that 😄 You can read more about template literals in JavaScript here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals